api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/context/MessageContextLoader.java

```diff
@@ -42,25 +42,31 @@ public DetermineEntitlementMessageContext loadContext(DetermineEntitlementMessag
                 .build();
     }
 
-    private Claim getAndCheckClaim(DetermineEntitlementMessagePayload payload) {
-        UUID claimId = payload.getClaimId();
+    /**
+     * Method used to inflate the contents of the objects identified by ids in the MAKE_PAYMENT message payload.
+     *
+     * @param payload The payload containing the object ids
+     * @return A wrapper object with the inflated objects
+     */
+    public MakePaymentMessageContext loadContext(MakePaymentMessagePayload payload) {
+        PaymentCycle paymentCycle = getAndCheckPaymentCycle(payload.getPaymentCycleId(), "payment cycle");
+        Claim claim = getAndCheckClaim(payload.getClaimId());
+
+        return MakePaymentMessageContext.builder()
+                .cardAccountId(payload.getCardAccountId())
+                .claim(claim)
+                .paymentCycle(paymentCycle)
+                .build();
+    }
+
+    private Claim getAndCheckClaim(UUID claimId) {
         Optional<Claim> claim = claimRepository.findById(claimId);
         if (claim.isEmpty()) {
             logAndThrowException("claim", claimId);
         }
         return claim.get();
     }
 
-    private PaymentCycle getAndCheckCurrentPaymentCycle(DetermineEntitlementMessagePayload payload) {
```

Comment:
I removed these methods as to do this for all message types would result in quite a lot of methods